### DISCUSSION TITLE
AudioGridderServer10.7_1.1_beta10 Install on  Mac OS Lion 10.7.4/5

### DISCUSSION THREAD

**Discussion by cyberishh** [apohl79/audiogridder#293] [Help]
On 10.7.4 the installer reports "fail to install" and on 10.7.5 it installs but AG crashes: (running LION in 32bit)

Process:         AudioGridderServer [775]
Path:            /Applications/AudioGridderServer.app/Contents/MacOS/AudioGridderServer
Identifier:      com.e47.AudioGridderServer
Version:         1.1.0 (1.1.0)
Code Type:       X86-64 (Native)
Parent Process:  launchd [658]

Date/Time:       2021-01-02 11:42:13.640 +0100
OS Version:      Mac OS X 10.7.5 (11G63)
Report Version:  9

Interval Since Last Report:          72444 sec
Crashes Since Last Report:           5862
Per-App Crashes Since Last Report:   2
Anonymous UUID:                      86D41F9C-8CFA-47D8-B395-A248D7148D4B

Crashed Thread:  0

Exception Type:  EXC_BREAKPOINT (SIGTRAP)
Exception Codes: 0x0000000000000002, 0x0000000000000000

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: /usr/local/opt/ffmpeg/lib/libavdevice.58.dylib
  Referenced from: /Applications/AudioGridderServer.app/Contents/MacOS/AudioGridderServer
  Reason: image not found

Binary Images:
       0x103efe000 -        0x104228fff +com.e47.AudioGridderServer (1.1.0 - 1.1.0) <7A116332-F481-3B15-B1F5-939A8A0D636A> /Applications/AudioGridderServer.app/Contents/MacOS/AudioGridderServer
    0x7fff63afe000 -     0x7fff63b32baf  dyld (195.6 - ???) <C58DAD8A-4B00-3676-8637-93D6FDE73147> /usr/lib/dyld

Model: MacPro3,1, BootROM MP31.006C.B05, 8 processors, Quad-Core Intel Xeon, 2.8 GHz, 10 GB, SMC 1.25f4
Graphics: NVIDIA GeForce GTX 285, NVIDIA GeForce GTX 285, PCIe, 1024 MB

**Comment by apohl79** [comment]
hmm it fails to load an ffmpeg lib, which it should not even try, as it should be linked statically. 

I’m reworking the build system right now and will have another look.

**Reply by cyberishh** [reply]
cool, looking forward to it! Thank you!
